app/src/main/kotlin/org/fossify/filemanager/activities/SaveAsActivity.kt

```diff
@@ -15,10 +15,24 @@ import java.io.File
 class SaveAsActivity : SimpleActivity() {
     private val binding by viewBinding(ActivitySaveAsBinding::inflate)
 
-    override fun onCreate(savedInstanceState: Bundle?) {
+
+        override fun onCreate(savedInstanceState: Bundle?) {
```

Comment:
Unnecessary indent here. 
```suggestion
    override fun onCreate(savedInstanceState: Bundle?) {
```